_posts/2023-07-23-i-lost-my-domain.md

```diff
@@ -6,10 +6,10 @@ tags: blogging domains blog
 categories: blog-meta
 ---
 
-Last year, during a particularly hectic period, I neglected my email inbox and, regrettably, overlooked several warnings from my Domain Registrar regarding the impending expiration of my domain, *brendonbody.com*. This domain has been mine for over 10 years, and while it had expired once before, I was always able to buy it back. This time, however, a [domain squatter](https://www.godaddy.com/resources/skills/what-is-domain-squatting-and-what-can-you-do-about-it) seized the opportunity and snapped it up.
+Last year, during a particularly hectic period, I neglected my email inbox and regrettably overlooked several warnings from my domain registrar about the impending expiration of my domain, *brendonbody.com*. I had owned this domain for over 10 years, and while it had expired once before, I was always able to reclaim it. This time, however, a [domain squatter](https://www.godaddy.com/resources/skills/what-is-domain-squatting-and-what-can-you-do-about-it) seized the opportunity and snapped it up.
 
 Faced with a minimum bid of $500 USD to reclaim my domain, I seriously contemplated whether maintaining a blog was worth it.
 
-Despite the considerable gaps between my posts, and despite [accidentally deleting my previous blog]({{site.baseurl}}/2016/02/21/hello-world-again/), I realized that I genuinely enjoy the process of blogging. I write for my own personal satisfaction, not for self-marketing purposes, and I've even stopped paying attention to my analytics. So the loss of incoming links and search engine ranking didn't bother me. I don't have any plan to write X posts over Y time period, I will just continue blogging when I feel like I want to say something.
+Despite the considerable gaps between my posts and [accidentally deleting my previous blog]({{site.baseurl}}/2016/02/21/hello-world-again/), I realized that I genuinely enjoy the process of blogging. I write for personal satisfaction rather than self-marketing, and I've even stopped monitoring my analytics. The loss of incoming links and search engine rankings didn't bother me. I don't have a plan to write X posts over Y time period; I simply blog when I have something to say.
```

Comment:
```suggestion
Despite the considerable gaps between my posts and despite [accidentally deleting my previous blog]({{site.baseurl}}/2016/02/21/hello-world-again/), I realized that I genuinely enjoy the process of blogging. I write for personal satisfaction rather than self-marketing, and I've even stopped monitoring my analytics. The loss of incoming links and search engine rankings didn't bother me. I don't have a plan to write X posts over Y time period; I simply blog when I have something to say.
```